Dolapo47/Auto-Mart
# implement changes on the signup page

#### What does this PR do?
- A summary of what this task is about
This PR enables user to navigate from the signup page
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
enable user redirect to dashboard when all inputs are filled
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
Navigate to the signup page
Input the relevant fields
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[story_id](https://www.pivotaltracker.com/story/show/story_id)`)
